ApiBundle/Controller/ControllerTrait/HandleRequestDataTable.php

```diff
@@ -5,6 +5,7 @@
 use Doctrine\ODM\MongoDB\DocumentRepository;
 use OpenOrchestra\BaseApi\Transformer\TransformerInterface;
 use Symfony\Component\HttpFoundation\Request;
+use OpenOrchestra\BaseApi\Facade\FacadeInterface;
```

Comment:
the publisher doesn't know the type of the return value and so the functionality (detect deprecated, type, etc.) provided by is lost